src/interfaces/PaymentSource.ts

```diff
@@ -0,0 +1,9 @@
+export interface IPaymentSource {
```

Comment:
@igorbezsmertnyi we don't have a PaymnentSource model - it's either CreditCard or StoreCredit

https://github.com/spree/spree/blob/master/api/app/serializers/spree/v2/storefront/credit_card_serializer.rb
https://github.com/spree/spree/blob/master/api/app/serializers/spree/v2/storefront/store_credit_serializer.rb